SolizWilson/colecciones/src/main.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Scanner;
+
+public class main {
+
+    private static int i;
+
+    public static void main(String[] args) {
+
+        Scanner leer = new Scanner(System.in);
+
+
+        Cliente a[] = new Cliente[2];
+
+        int codigo= 0;
+        String nombres = "";
+        int telefono = 0;
+        String direccion = "";
+        String descripcion = "";
+
+        for (int i = 0;i<a.length;i++);
```

Comment:
watch out with the syntax of the for loop it doesn't require a semicolon after the closing parenthesis.